File: 1319-unique-number-of-occurrences/unique-number-of-occurrences.swift
class Solution {
    func uniqueOccurrences(_ arr: [Int]) -> Bool {
        var counts: [Int:Int] = [ : ]

        for number in arr {
            counts[number, default: 0] += 1
        }


        let values = counts.values
        return Set(values).count == values.count

    }
}

File: 1319-unique-number-of-occurrences/unique-number-of-occurrences.py
class Solution:
    def uniqueOccurrences(self, arr: List[int]) -> bool:
        dict = {}

        for i in arr:
            dict[i] = dict.get(i, 0) + 1
        

        values = dict.values()

        return len(set(values)) == len(values)




        